build.gradle

```diff
@@ -44,18 +46,26 @@ allprojects {
         detektPlugins "io.gitlab.arturbosch.detekt:detekt-formatting:${Versions.detekt}"
     }
 }
-
-tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
+tasks.withType(KotlinCompile).configureEach {
     kotlinOptions {
         jvmTarget "1.8"
     }
 }
 
 detekt {
     toolVersion = Versions.detekt
-    input = files("src/main/kotlin")
-    filters = ".*/resources/.*,.*/build/.*"
+    input = files(rootProject.rootDir)
+    buildUponDefaultConfig = true
     config = files("detekt.yml")
+    reports {
+
+        xml {
+            enabled = true
+        }
+        html {
```

Comment:
I think the HTML is enough. Less Build Time 😜

Reply:
done